### DISCUSSION TITLE
Townhall meeting (7 Dec 2022) Crazyradio 2.0 and comstack

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#426] [Announcements]
We will have a town hall meeting about the new communication protocol and Crazyradio 2.0. It will be about this blogpost we recently published: https://www.bitcraze.io/2022/11/new-crazyradio-and-new-communication-architecture/.

 Please come if you'd like to discuss the proposed changes and have any questions.

Date: Wednesday 7th of December
Time: 3 pm, 15:00 Central European time. 

**Meeting link**: Meeting has already passed

Also give a 👍  if you are planning to join :)

**Comment by cubicrootk** [comment]
👍

**Comment by knmcguire** [comment]
Please read the blogpost as well! https://www.bitcraze.io/2022/11/new-crazyradio-and-new-communication-architecture/
